pkg/process/net/common_windows.go

```diff
@@ -38,6 +39,8 @@ const (
 	conntrackCachedURL = "http://localhost:3333/" + string(sysconfig.NetworkTracerModule) + "/debug/conntrack/cached"
 	// conntrackHostURL is not used on Windows, the value is added to avoid a compilation error
 	conntrackHostURL = "http://localhost:3333/" + string(sysconfig.NetworkTracerModule) + "/debug/conntrack/host"
+	// ebpfBtfLoaderURL is not used on Windows, the value is added to avoid a compilation error
+	ebpfBTFLoaderURL = ""
```

Comment:
is this needed? I only see it referenced in `common_linux.go`

Reply:
Ah you are right! Will remove

Reply:
Removed 👍